src/screens/OtherProfile/index.js

```diff
@@ -642,9 +642,12 @@ const OtherProfile = () => {
           extraData={[feeds]}
           data={handleDataFeed()}
           onScroll={handleScroll}
-          onEndReach={__handleOnEndReached}
           onRefresh={handleRefresh}
           refreshing={loading}
+          onMomentumScrollEnd={(event) => {
+            onWillSendViewPostTime(event, feeds);
+            fetchNextFeeds(event);
+          }}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of the `onMomentumScrollEnd` event handler is a good change as it allows for more efficient loading of feeds and better tracking of post view times. However, ensure that the `onWillSendViewPostTime` and `fetchNextFeeds` functions handle potential errors properly to avoid crashes or unexpected behavior.


<!-- This is an auto-generated comment by OSS CodeRabbit -->